scrapers/Brazzers.yml

```diff
@@ -1,19 +1,42 @@
-name: 'brazzers'
+name: 'Brazzers'
 sceneByURL:
   - action: scrapeXPath
     url:
       - brazzers.com/video
       - brazzers.com/scenes
+      - men.com/sceneid/
+      - realitykings.com/scene/
     scraper: sceneScraper
+
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - brazzers.com/pornstar
+    scraper: performerScraper
+
+performerByName:
+  action: scrapeXPath
+  queryURL: https://www.brazzers.com/pornstars?q={}
+  scraper: performerSearch
+
 xPathScrapers:
+  performerSearch:
+    common:
+      $result: //span/a[contains(@href, "/pornstar")]
+
+    performer:
+      Name: $result/@title
+      URL:
+        selector: $result/@href
+        replace:
+          - regex: ^
+            with: https://www.brazzers.com
+
   sceneScraper:
     common:
       $titlebar: //*[name()='path'][1]/../../../../h2/..
     scene:
       Title: $titlebar/h2/text()
-      Studio:
-        Name: 
-          selector: //div[text()="Subsite"]/../a/text()
 # Remove '#' if you want get the studio without space.
```

Comment:
You forgot to remove the comments for the studio

Also it seems that we can get the studio by getting the `//link[@rel="canonical"]/@href` and doing a regex replace

Reply:
It was a mistake when i added the comment...
`//div[text()="Subsite"]/../a/text()` give the series which is better (Like PornDB Stash Python do)
`//link[@rel="canonical"]/@href` work but give only Brazzers, RealityKings, Men